cfgov/v1/tests/test_util_migrations.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import mock
+from unittest import TestCase
+
+from v1.util.migrations import (
+    migrate_page_types_and_fields,
+    migrate_stream_field,
+    get_stream_data,
+    set_stream_data,
+)
+
+
+class MigrationsUtilTestCase(TestCase):
```

Comment:
👍  for creating unit tests, but these all seem to run the tests against `Mock` objects. Is it possible to write unit tests that actually run these tests against real `Page` and `PageRevision` objects?